src/NServiceBus.AwsLambda.SQS/AwsLambdaEndpoint.cs

```diff
@@ -44,7 +45,7 @@ await InitializeEndpointIfNecessary(lambdaContext, cancellationToken)
 
             foreach (var receivedMessage in @event.Records)
             {
-                processTasks.Add(ProcessMessage(receivedMessage, lambdaContext, cancellationToken));
+                processTasks.Add(ProcessMessage(receivedMessage.ToMessage(), receivedMessage, lambdaContext, cancellationToken));
```

Comment:
can we call ToMessage inside ProcessMessage to not pass two different message arguments to the method?